build/package/download-msi.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# Copyright 2021 MongoDB Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -Eeou pipefail
+
+VERSION=$(git describe | cut -d "v" -f 2)
+
+curl https://fastdl.mongodb.org/mongocli/mongocli_"${VERSION}"_windows_x86_64.zip --output "${msi_file_name:?}".msi
```

Comment:
[q] the zip contains a binary, the `msi` is just an `msi`, does this really work?

Reply:
ops this is a left over